marathon_acme/tests/fake_marathon.py

```diff
@@ -44,8 +48,22 @@ def get_event_subscriptions(self):
         return self._event_subscriptions
 
     def add_event_subscription(self, callback_url):
-        assert callback_url not in self._event_subscriptions
-        self._event_subscriptions.append(callback_url)
+        if callback_url not in self._event_subscriptions:
+            self._event_subscriptions.append(callback_url)
+
+    def trigger_event(self, event_type, **kwargs):
+        event = {
+            'eventType': event_type,
+            # FIXME: This isn't quite like Marathon's timestamp.
+            # Differences: this has microseconds instead of milliseconds,
+            # Marathon has a trailing 'Z'.
+            'timestamp': datetime.utcnow().isoformat()
```

Comment:
Ah, right. I think what you have there is okay, but maybe wrap it in a function.